packages/docs-app/src/examples/core-examples/nonIdealStateExample.tsx

```diff
@@ -126,6 +126,7 @@ export class NonIdealStateExample extends React.PureComponent<IExampleProps, INo
                     description={this.state.showDescription ? description : undefined}
                     action={this.state.showAction ? action : undefined}
                     layout={this.state.layout}
+                    role="alert"
```

Comment:
this example does not show an [urgent, time-sensitive message](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/alert_role), so this role is inappropriate.

to be honest I don't see any role in the [list here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles) which would be appropriate for a NonIdealState. so that begs the question, why are we adding this feature or demonstrating a `<NonIdealState>` example using the `role` attribute

Reply:
we can just cancel this PR